src/App.js

```diff
@@ -1,12 +1,25 @@
 import React, { Component } from 'react';
+import './app.scss';
+import { Button } from 'carbon-components-react';
+import { Content } from 'carbon-components-react/lib/components/UIShell';
+import TutorialHeader from './components/TutorialHeader';
 
+import { Route, Switch } from 'react-router-dom';
+import LandingPage from './content/LandingPage';
+import RepoPage from './content/RepoPage';
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello Carbon! Well, not quite yet. This is the starting point for the
-        Carbon tutorial.
-      </div>
+      <>
+        <TutorialHeader />
+        <Content>
+          <Switch>
+            <Route exact path="/" component={LandingPage} />
+            <Route path="/repos" component={RepoPage} />
+          </Switch>
+          <Button>Button</Button>
```

Comment:
Per https://www.carbondesignsystem.com/tutorial/react-step-1#add-routing, the `<Button>` component should be removed.